### DISCUSSION TITLE
I need to use a lot of subdomains (10000 +), next.js Can you satisfy my needs?

### DISCUSSION THREAD

**Discussion by x751685875** [vercel/next.js#24210] [Help]
I need to use a lot of subdomains (10000 +), next.js Can you satisfy my needs? How do I get my subdomain name? How to match rendering JS？

a.domain.com
b.domain.com
....domain.com

**Comment by x751685875** [comment]
Can someone answer me？

**Reply by apollonian** [reply]
If I understand correctly, you are looking to read subdomains and handle things accordingly? In that case, you could use `getServerSideProps` and read the `host` header in the incoming request.
